charmcraft/commands/init.py

```diff
@@ -79,13 +79,16 @@ def fill_parser(self, parser):
             "--series", default="kubernetes",
             help="A comma-separated list of supported platform series;"
             " defaults to 'kubernetes'")
+        parser.add_argument(
+            "-f", "--force", type="store_true",
+            help="Initialize even if the project dir is not empty. Will not overwrite files")
```

Comment:
As we're not finishing with dots, it looks weird to have two sentences. What about "Initialize even if the directory is not empty (will not overwrite files)"